src/calculateRentalCost.js

```diff
@@ -4,7 +4,23 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const PRICE_PER_DAY = 40;
+  const LONG_TERM = 7;
+  const LONG_TERM_DISCOUNT = 50;
+  const SHORT_TERM = 3;
+  const SHORT_TERM_DISCOUNT = 20;
+
+  const basePrice = days * PRICE_PER_DAY;
+
+  if (days >= LONG_TERM) {
+    return basePrice - LONG_TERM_DISCOUNT;
+  }
+
+  if (days >= SHORT_TERM) {
+    return basePrice - SHORT_TERM_DISCOUNT;
+  }
```

Comment:
The logic for applying discounts is sequential, meaning if the number of days is greater than or equal to `LONG_TERM`, it will apply the `LONG_TERM_DISCOUNT` and skip checking the `SHORT_TERM` condition. If the intention is to apply both discounts when both conditions are met, you need to adjust the logic to accumulate discounts rather than returning immediately.